blueocean-core-js/src/less/run-button.less

```diff
@@ -32,17 +32,9 @@
         }
 
         .stop-button {
-            padding: 2px;
-
-            .btn-icon {
-                width: 20px;
-                height: 20px;
-                background-image: url('./icons/stop-button-blue.svg');
-
-                &:hover {
-                    background-image: url('./icons/stop-button-blue-dark.svg');
-                }
-            }
+            /* asset has different padding than material-icon asset used in .run-button: nudge it slightly */
```

Comment:
Stop button now uses a SVG inline to again avoid the need for 4 different external SVG assets. The SVG is tiny, so it shouldn't be bad at all.